p2pspec

```diff
@@ -1,4 +1,36 @@
-:::NEW p2p spec:::
+:::POTATO MESSAGES:::
+
+:establishing connection:
+- A->B connect request [conn_uuid, challenge, identity {A, B}]
+  - spam when initiating connection
+  - retry every 10 minutes
+- B->A connect response [conn_uuid, B signed challenge, challenge]
+  - B now stores {conn_uuid, A} in active connections
+- A->B connect response response [conn_uuid, A signed challenge]
+  - A now stores {conn_uuid, B} in active connections
+- A<->B ping
+  - periodic message to indicate the connection is still alive
+
```

Comment:
do we need any of this?

we can do without ping

we also don't need any handshake; anyone can connect

every sent message will be signed